scripts/circle-ci/pr-changelog-status-check.sh

```diff
@@ -4,23 +4,39 @@ set -x
 
 CHANGELOG="docs/source/release_notes/release-notes.rst"
 
-if [ -z "$GITHUB_AUTH_TOKEN" ];
-then
-    exit 1
-fi
-
 cd $(dirname $0)
 
 CURRENT_REF=$(git log -1 --format="%H")
 CHANGELOG_COMMIT_FLAG=$(git log --pretty=format:%B origin/develop..HEAD | grep -iq '\[no release notes\]')$?
 CHANGELOG_MODIFIED=$(git log --name-only --pretty=format: origin/develop..HEAD | grep -q $CHANGELOG)$?
 
+success() {
+    message=$1
+    if [ -z "$GITHUB_AUTH_TOKEN" ]; then
+        echo "$message"
+    else
+        curl -X POST -f --silent --header "Authorization: token $GITHUB_AUTH_TOKEN" -d '{"state": "success", "description": "'"$message"'", "context": "changelog"}' "https://api.github.com/repos/palantir/atlasdb/statuses/$CURRENT_REF" >/dev/null
```

Comment:
OK - I have verified that this (changed in commit 3) string-substitution works as expected.